src/test/kotlin/com/terraformation/backend/seedbank/model/accession/AccessionModelViabilityTest.kt

```diff
@@ -0,0 +1,420 @@
+package com.terraformation.backend.seedbank.model.accession
+
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.seedbank.ViabilityTestId
+import com.terraformation.backend.db.seedbank.ViabilityTestType
+import com.terraformation.backend.db.seedbank.WithdrawalPurpose
+import com.terraformation.backend.seedbank.grams
+import com.terraformation.backend.seedbank.model.ViabilityTestModel
+import com.terraformation.backend.seedbank.seeds
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class AccessionModelViabilityTest {
```

Comment:
This is a tough one to name! Either `Test` does double duty as the suffix of the unit test class name and part of the name of the thing being tested, or you have `TestTest` which looks weird. (The choice you made here is fine.)

Reply:
I almost did the `TestTest` thing, or `AccessionModelViabilityTestCalculationsTest` but that was getting awfully long 😂